### PR TITLE
Add aicspylibczi package

### PR DESCRIPTION
**Pull request by multimeric**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/aicspylibczi/build.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export CMAKE_GENERATOR=Ninja
-{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+python3 -m pip install . -vv --no-deps --no-build-isolation
```
Comment (zklaus) [inline recipes/aicspylibczi/build.sh:2]:
```suggestion
${PYTHON} -m pip install . -vv --no-deps --no-build-isolation
```

Comment (zklaus) [reply recipes/aicspylibczi/build.sh:2]: You are right that you cannot use jinja templating. But you can (and should) use some environment variables (c.f. https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html).

File: recipes/aicspylibczi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+  sha256: 435d1544421cacc05838aa31afaade85da0972bd3638bfe008fae6e023248ad0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (zklaus) [inline recipes/aicspylibczi/meta.yaml:17]:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

**Comment by multimeric** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/aicspylibczi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools >=42
+    - cmake >=3.18
+    - ninja
```
Comment (xhochy) [inline recipes/aicspylibczi/meta.yaml:24]: Ninja and cmake should be in build

Comment (multimeric) [reply recipes/aicspylibczi/meta.yaml:24]: Hmm, that part was generated by `grayskull`, I wonder why it did that.

**Comment by multimeric** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Review by xhochy — APPROVED** [review]

**Pull request merged**